packages/profile-sync-controller/src/controllers/user-storage/UserStorageController.ts

```diff
@@ -220,6 +359,9 @@ export default class UserStorageController extends BaseController<
   constructor(params: {
     messenger: UserStorageControllerMessenger;
     state?: UserStorageControllerState;
+    env?: {
+      isAccountSyncingEnabled?: boolean;
+    };
```

Comment:
Perfect!

I prefer this as we can then allow platforms to control this (for worst case of easy rollback)